staging/src/k8s.io/apiserver/pkg/authorization/authorizer/interfaces.go

```diff
@@ -144,3 +144,11 @@ func (a AttributesRecord) IsResourceRequest() bool {
 func (a AttributesRecord) GetPath() string {
 	return a.Path
 }
+
+type Decision int
+
+const (
+	DecisionDeny Decision = iota
+	DecisionAllow
+	DecisionNoOpinion
```

Comment:
Add documentation here about what each decision means. In particular, NoOpinion means "default deny".